tests/cucumber.conf.js

```diff
@@ -7,7 +7,7 @@ const util = require('util')
 const exec = util.promisify(require('child_process').exec);
 
 const tandoorURL = process.env.APP_URL || "http://localhost/"
-
+const imagePath="filesForUpload/"
```

Comment:
use formatter. if using vscode, install https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode
```suggestion
const imagePath = "filesForUpload/"
```